basic/.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   test:
     docker:
-      - image: cypress/base:10
```

Comment:
Cypress v12 dropped node 12 support. This is an example for users. Updated to pull latest.